_projects/rct.md

```diff
@@ -5,9 +5,25 @@ collection: projects
 permalink: /projects/rct
 abstract: "Scalable, Interoperable and Sustainable Tools for Science: Supporting scalable, interoperable and sustainable science on a range of high-performance and distributed computing infrastructures."
 status: active
+logo: rct_logo.png
 ---
 
-RADICAL Cybertools (RCT) are an abstractions-based suite of software modules designed to support scalable, interoperable and sustainable science on a range of high-performance and distributed computing infrastructure. RCT build upon important theoretical advances, software development best practices, and carefully-analyzed usage and programming models. RCT play a role in grand-challenge problems, ranging from personalized medicine and health to understanding long-term global and regional climate. All software developed through the project is open source, licensed under the MIT License (MIT), and available on <a href="https://github.com/radical-cybertools">GitHub</a>. The four main RCT modules are: <a href="https://github.com/radical-cybertools/saga-python">RADICAL-SAGA (RS)</a>, <a href="https://github.com/radical-cybertools/radical.pilot">RADICAL-Pilot (RP)</a>, <a href="https://github.com/radical-cybertools/radical.entk">RADICAL Ensemble Toolkit (EnTK)</a>, and <a href="https://github.com/radical-cybertools/radical.analytics">RADICAL-Analytics (RA)</a>.
-
-<a href="https://radical-cybertools.github.io/"><i></i>Website</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1440677"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/rct_logo.png" width="150">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```